setup.py

```diff
@@ -94,7 +94,17 @@ def fix_url_dependencies(req: str) -> str:
     author="Allen Institute for Artificial Intelligence",
     author_email="allennlp@allenai.org",
     license="Apache",
-    packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"],),
+    packages=find_namespace_packages(
```

Comment:
Can't we just include what we want, instead of excluding what we don't want?

Also, this will treat the regular package as a "namespace package" as well. Is that acceptable?

Reply:
Using include led to that bug before, so I’m trying to avoid it since the behavior is not consistent across environments. And I don’t think the main package is treated any differently, at least from what I could tell through building and installing it this way